eslint.config.mjs

```diff
@@ -0,0 +1,59 @@
+import Codex from "eslint-config-codex";
+import { plugin as TsPlugin, parser as TsParser } from 'typescript-eslint';
+
+export default [
+    ...Codex,
+    {
+        files: ['src/**/*.ts'],
+        languageOptions: {
+            parser: TsParser,
+            parserOptions: {
+                project: './tsconfig.json',
+                tsconfigRootDir: './',
+                sourceType: 'module',
+            },
+        },
+        rules: {
+            'n/no-missing-import': ['off'],
+            'n/no-unsupported-features/node-builtins': ['off'],
+            'jsdoc/require-returns-description': ['off'],
+            'jsdoc/require-jsdoc': [
```

Comment:
please, explain the reason of this rule override

Reply:
All the eslint rules that you are seeing are taken from image repo and I have modified only one or two rules.

https://github.com/editor-js/image/blob/master/eslint.config.mjs

Reply:
It does not change the `require-jsdoc` rule

Reply:
some methods with explicit return type in function signature were throwing `require-jsdoc` error. So I fixed it with this rule. 
Now I will be removing this rule and fix the return type with alias return type.

Reply:
you can disable `jsdoc/require-returns-type` for ts files (it should be done in codex config, correct me if it is not working)

Reply:
I think its not necessary as of now as the rule is not required and it has will be removed in the next commit.